src/openvic-simulation/country/CountryInstance.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "openvic-simulation/country/Country.hpp"
+#include "openvic-simulation/history/CountryHistory.hpp"
+#include "openvic-simulation/map/Province.hpp"
+#include "openvic-simulation/utility/Getters.hpp"
+
+namespace OpenVic {
+    /* Representation of an existing country that is currently in-game. */
+    struct CountryInstance {
+    private:
+        Country const* PROPERTY_RW(base_country)
+        Culture const* PROPERTY_RW(primary_culture)
+        std::vector<Culture const*> PROPERTY(accepted_cultures)
+        Religion const* PROPERTY_RW(religion)
+        CountryParty const* PROPERTY_RW(ruling_party)
+        Date PROPERTY_RW(last_election)
+        decimal_map_t<Ideology const*> PROPERTY(upper_house)
+        Province const* PROPERTY_RW(capital)
+        GovernmentType const* PROPERTY_RW(government_type)
+        fixed_point_t PROPERTY_RW(plurality)
+        NationalValue const* PROPERTY_RW(national_value)
+        bool PROPERTY_RW(civilised)
+        fixed_point_t PROPERTY_RW(prestige)
+        std::vector<Reform const*> PROPERTY(reforms)
```

Comment:
This works for now as a place to put the country history reforms, eventually we'll need a map from reform groups to single active reforms from that group.